elastic-stack/templates/redis-input.conf

```diff
@@ -7,6 +7,6 @@ input {
     db => 0
 
     # Redis removes @metadata and we need it in the output
-    add_field => { "[@metadata][beat]" => "packetbeat" }
+    add_field => { "[@metadata][index]" => "packetbeat" }
```

Comment:
I'm not sure yet how this works with the syslog pipeline... Does beats now use `index` or `beat`?

Reply:
Related issue: https://github.com/elastic/beats/issues/7389